release-notes.txt

```diff
@@ -1,3 +1,565 @@
++ drivers/sen5x: Add device driver for SEN5x (#19955)
++ drivers/tm1637: add driver and tests for tm1637 7-segment display (#21112)
+* driver/sen5x: Small Fixups (#21392)
+* drivers/at: at_readline_stop_at_str fix remaining buffer length (#21195)
+* drivers/sdmmc/sdmmc_sdhc: remove DMA check (#21260)
+* drivers/{mtd_sdmmc,sdmmc}: reinit card (#21305)
+
+Documentation
+-------------
+
++ add *.doc.md to doxygen sources (#21319)
++ boards/nrf52840{dk,dongle}: add MCU table, move to doc.md (#21271)
++ boards/nucleo-f413zh: add MCU table, move to doc.md (#21264)
++ dist/tools: Add Doxygen as a Tool for CI (#21300)
++ doc, buildsystem: Add a Version Check for Doxygen, Fix doc* Target
+  Execution (#21277)
++ doc/managing-a-release: add looking out for release tests to preps (#21368)
++ doc: add documentation on RIOT_FILE_RELATIVE (#21170)
++ doc: add psutil requirement to  Getting Started (#21399)
+* boards/arduino-due: update doc.txt (#21207)
+* boards/nucleo-144: rename doc.txt to doc.md (#21318)
+* boards/nucleo-f746zg: board doc update (#21310)
+* boards/rpi-pico*: Update and fix documentation and image (#21218)
+* build-system: Drop -Wno-error=documentation (#21377)
+* doc/porting-boards: Change example doc.txt to doc.md (#21257)
+* doc: doc.txt -> doc.md (#21255)
+* doc: hack to hide doc.md files until doxygen gets fixed (#21273)
+* doc: remove and replace deprecated mailing lists (#21285)
+* doc: Use the correct Doxygen Logo File Type in the Footer (#21274)
+* docs/rust: Install c2rust with --locked (#21236)
+* docs: Switch out pkg reference link (#21229)
+* sys/cord/doc.txt: Update additional references to RFC 9176 (#21159)
+* sys/malloc_monitor: Fix documentation (#21225)
+* sys/psa_crypto/doc: correctly end code block (#21380)
+* treewide: fix double the in doc and comments (#21224)
+* treewide: fix example references in docs (#21212)
+* CODING_CONVENTION.md: migrate to `#pragma once` (#21369)
+
+Build System / Tooling
+----------------------
+
++ buildsystem: Enable -Wdocumentation for LLVM (#21358)
++ dist/tools: add `#pragma once` to headerguard check (#21367)
+* buildsystem: Remove `buildtest` Goal & Fix `info-buildsizes-diff` (#21359)
+* dist/tools/bmp: additional improvements to auto-detection (#21128)
+* dist: tools: avoid hanging tapsetup in the absence of a DHCP server (#20411)
+* Makefile.include: perform checks for all relevant targets (#21199)
+* makefiles: bump riotdocker (#21258)
+* makefiles: colorize board alias warning (#21352)
+* makefiles: fix output of info-applications (#21213)
+* tools: Use RIOT maintained cosy (#21070)
+
+
+Examples
+--------
+
+* examples: restructure examples folder (#21135)
+* examples: shorten subfolders' names (#21221)
+* Configure default DNS resolver in 6lbr example (#21117)
+* examples/nanocoap_server: enable LWIP IPv4 (#21333)
+* examples: migrate to XFA SHELL_COMMAND (#21171)
+* examples: update `dtls-echo` source (#21247)
+* Request temporary address (IA_NA) for the 6lbr example (#21118)
```

Comment:
```suggestion
* examples/gnrc_border_router: Request temporary address (IA_NA) for the 6lbr example (#21118)
```

I was so free to already update the PR title in https://github.com/RIOT-OS/RIOT/pull/21118